core/dictgen/res/rootcling_impl.h

```diff
@@ -41,11 +41,13 @@ namespace RootCling {
 } // namespace Internal
 } // namespace ROOT
 
+#ifndef R__DLLEXPORT
```

Comment:
Where else is `R__DLLEXPORT` defined? Shouldn't we be including that header instead?

Reply:
> Where else is R__DLLEXPORTdefined

There is no special include for that. 
Just in approx 5 places same kind of definitions is provided.